tapp.gemspec

```
@@ -20,5 +20,5 @@ Gem::Specification.new do |s|
 
   s.add_runtime_dependency 'thor'
 
-  s.add_development_dependency 'turnip'
+  s.add_development_dependency 'turnip', '~> 3'
```

Can you add a reason why the version is pinned?

```suggestion
  # Pin a Turnip version that doesn't use Ruby 2.3 safe navigation operator syntax.
  s.add_development_dependency 'turnip', '~> 3'
```